frontend/src/queries/nodes/PersonsNode/PersonsSearch.tsx

```diff
@@ -20,8 +20,7 @@ interface LabelType {
 const labels: Record<ActorType, LabelType> = {
     person: {
         label: 'persons',
-        description:
```

Comment:
I think this is still in reference to the legacy implementation. The current query looks like this:

```sql
SELECT id, id, created_at, 1
FROM persons
WHERE or(
    ilike(properties.email, '%QUERY%'), 
    ilike(properties.name, '%QUERY%'), 
    ilike(toString(id), '%QUERY%'), 
    in(id, (SELECT person_id FROM person_distinct_ids WHERE ilike(distinct_id, '%QUERY%')))
)
ORDER BY created_at DESC
LIMIT 101 OFFSET 0
```